test/generators.test.js

```diff
@@ -335,6 +335,7 @@ describe('Generators', function () {
         var tags = generators.tags(models);
         var generatedRoutes = generators.routes(controllers, {routes: {'get /user/phone/:phoneNumber': 'UserController.phone'}}, tags);
         var actual = generators.paths(generatedRoutes, tags, {list: [{$ref: '#/parameters/PerPageQueryParam'}]});
+        console.log ('>>',generatedRoutes);
```

Comment:
no need for this

Reply:
true

Reply:
you can add a nice test case and pass `config.blueprint` props as the configuration for generator.routes function here (check the way `routes` object was passed in for `routes.js`)
 https://github.com/theo4u/sails-hook-swagger-generator/blob/a9d9862ba3db79076424749721fe4390ee672c22/test/generators.test.js#L163